% Chapter 02
\chapter{Parent of Origin Effects on Quantitative Phenotypes in a Founder Population}\label{ch:pogwas}
\section[Abstract]{Abstract\footnotemark}


The impact of the parental origin of associated alleles in GWAS has been largely ignored. Yet sequence variants could affect traits differently depending on whether they are inherited from the mother or the father. To explore this possibility, we studied 21 quantitative phenotypes in a large Hutterite pedigree. We first identified variants with significant single parent (maternal-only or paternal-only) effects, and then used a novel statistical model to identify variants with opposite parental effects. Overall, we identified parent of origin effects (POEs) on 11 phenotypes, most of which are risk factors for cardiovascular disease. Many of the loci with POEs have features of imprinted regions and many of the variants with POE are associated with the expression of nearby genes. Overall, our results indicate that POEs, which can be opposite in direction, are relatively common in humans, have potentially important clinical effects, and will be missed in traditional GWAS. 


\footnotetext{Citation for chapter: }

\section{Introduction}\label{fig-introduction}
Genome-wide association studies (GWAS) typically treat alleles inherited from the mother and the father as equivalent, although variants can affect traits differently depending on whether they are maternal or paternal in origin. In particular, parent of origin effects (POEs) can result from imprinting, where epigenetic modifications allows for differential gene expression on homologous chromosomes that is determined by the parental origin of the chromosome. Mutations in imprinted genes or regions can result in diseases. For example, two very different diseases, Prader-Willi Syndrome (PWS) and Angelman Syndrome (AS), are due to loss of function alleles in genes within an imprinted region on chromosome 15q11-13. Inheriting a loss of function mutation for the \emph{SNRPN} gene from the father results in PWS but inheriting a loss of function mutation for the \emph{UBE3A} gene from the mother results in AS\citep{Peters2014,Falls1999}.  Long noncoding RNA genes at this and other imprinted regions act to silence (i.e. imprint) genes in cis. Imprinted genes are often part of imprinted gene networks, suggesting regulatory links between these genes \cite{Patten:2016cb,Gabory:2009be,Varrault:2006kn}. More than 200 imprinted loci have been described in humans \cite{Benonisdottir:2016dz} but there are likely many other, as yet undiscovered, imprinted loci. 

Previous studies have utilized pedigrees to test maternal and paternal alleles separately for association with phenotypes or with gene expression to uncover new imprinted loci \citep{Kong:2009kk,Baran:2015cx,Garg2012a,Paper2014b,Benonisdottir:2016dz}. Kong \emph{et al} \citep{Kong:2009kk} discovered one locus associated with breast cancer risk only when the allele is inherited from the father and another locus associated with type 2 diabetes risk only when the allele is inherited from the mother. Garg et al. reported parent-of-origin cis-eQTLs with known or putative novel imprinted genes affecting gene expression\citep{Garg2012a}. Two additional studies by Zoledziewsk et al. and Benonisdottir et al. identified opposite POEs on adult height at known imprinted loci\citep{Zoledziewska:2015do,Benonisdottir:2016dz}. Both studies reported associations with variants at the \emph{KCNQ1} gene, and one showed additional opposite POEs with height at two known imprinted loci (\emph{IGF2-H19} and \emph{DLK1-MEG3})\citep{Benonisdottir:2016dz}These studies provide proof-of-principle that alleles at imprinted loci can show POEs, some with opposite effects, with common phenotypes. 

Many existing studies and methods identify parent of origin effects use case/parent trios or case/mother duos\citep{Chuang:2017kp,Howey:2012hj,Ainsworth:2010bp,Weinberg:1999km,Weinberg:1998cf}. Similar to Kong \emph{et al.} \citep{Kong:2009kk}, our method does not require data on the parent and only uses the parent of origin informative alleles which were assigned and phased using PRIMAL\citep{Livne2015}.  In contrast to Kong \emph{et al.} \citep{Kong:2009kk} which used binary traits, our method tests for parent of origin effects on quantitative traits, similar to Benonisdottir \emph{et al.} \citep{Benonisdottir:2016dz} which tested for parent of origin effects on height.

No previous study has included a broad range of human quantitative phenotypes or has studied genome-wide variants with effects in different directions depending on the parent of origin. To address this possibility, we developed a statistical model that directly compares the effects of the maternal and paternal alleles to identify effects that are different, including those that are opposite. We applied this model in a study of 21 common quantitative traits that were measured in the Hutterites, a founder population of European descent for which we have phased genotype data \citep{Livne2015} We identified variants with maternally inherited or paternally inherited effects only and variants with opposite POEs. Some of the identified regions have characteristics similar to known imprinted genes. Overall, we show that this model can identify putative novel imprinted regions with POEs for a broad range of clinically relevant quantitative phenotypes.

\section{Results}\label{fig-results}

\subsection{GWAS}\label{GWAS Results}
We first performed standard genome-wide association studies (GWAS) of 21 traits in the Hutterites (Table S1). These studies identified one genome wide significant association (p \textless 5x10-8) with each of five of the 21 traits: low density lipoprotein level (LDL)-cholesterol, triglycerides, carotid artery intima media thickness (CIMT), left ventricular mass index (LVMI), and monocyte count. The results of all 21 GWAS are summarized in Table S2 and Supplementary Figure \ref{fig:fig-s1a}. Results for all variants for all GWAS are deposited in dbGaP (phs000185 ? submission in progress).

\subsection{Parent of Origin GWAS}\label{Parent of Origin GWAS}
We considered two possible mechanisms of POEs. In the first, the effect size of one parent's allele is close to zero and the effect size of the other parent's allele is significantly different from zero. For these cases, we performed a paternal only or maternal only GWAS. In other cases, the maternal and paternal alleles may both have effect sizes different from zero, but the effects are significantly different from each other or opposite in direction. To detect these types of POEs, we developed a model that tests for differences between parental effects (see Methods). This model is especially powerful to identify variants with parental effects in opposite directions.

\subsection{Maternal and Paternal GWAS}\label{Maternal and Paternal GWAS}
Using the same phenotypes, genotypes, pedigree, and criteria for significance as in the standard GWAS, we tested for maternal and paternal effects on each trait by testing each parentally inherited allele with the trait of interest, similar to previous studies\citep{Kong:2009kk,Zoledziewska:2015do,Garg2012a}. Variants were considered to have POEs if they had a p-value less than 5x10-8 in only one parent and were not significant in the standard GWAS (i.e., the LDL association on chromosome 19 and the triglycerides association chromosome 11 were not considered to have POEs; see Table S1). The most significant parent of origin associations are summarized in Table \ref{tab:singleparent}. All significant results of the parent of origin GWAS for all 21 phenotypes are included in Table S5. 


\begin{table}
\centering
%\begin{adjustbox}{width=1\textwidth}
%\small
\begin{tabular}{@{}p{3cm}lp{1cm}lp{2cm}lp{2cm}lp{2cm}lp{2cm}lp{2cm}lp{2cm}lp{2cm}lp{2cm}lp{2cm}lp{2cm}@{}}
\toprule Phenotype & rsid & chr:loc & Variant Location & Nearest Gene & MAF & N & Beta (SE) & Paternal GWAS p-value & Maternal GWAS p-value & Standard GWAS p-value \\ \midrule
A. Maternal Associations &&&&&&&&&& \\
Age at Menarche  & rs7184983 (A/G) & 16:56554709 & Upstream & \emph{BBS2} & 0.059 & 336 & 0.862 (0.154) & 5.01e-01 & 3.11e-08 & 6.75e-03 \\ 
B. Paternal Associations &&&&&&&&&& \\ \bottomrule
\end{tabular}
%\end{adjustbox}
\caption[Phenotypes with significant single parent of origin associations. ]{\textbf{Phenotypes with significant single parent of origin associations.}
 *The most significant variant (P \textless 5x10?8) at each locus for the (A) maternal and (B) paternal associations associated with each phenotype is shown. 
*non-coding RNA genes}
\label{tab:singleparent}
\end{table}



Overall, seven phenotypes had genome-wide significant parent of origin associations: four in the maternal only GWAS and three in the paternal only GWAS. Three cardiovascular disease (CVD)-associated phenotypes (age at menarche, CIMT, LVMI) and one lung function phenotype (forced expiratory volume in one second [FEV1]) were associated with maternally-inherited alleles only. 

A maternally inherited allele at rs7184983 (G) on chromosome 16 was associated with younger age of menarche (P = 3.11 x10-8) (Figure \ref{fig:menarche_mpgwas}). This SNP, rs7184983, is located upstream of the \emph{BBS2} gene and is associated with increased expression of \emph{OGFOD1} in transformed fibroblast cells and tibial nerve \cite{Consortium2015}. The maternally inherited allele at rs4077567 (G) on chromosome 2 was associated with decreased CIMT (P = 3.02x10-8) (Figure \ref{fig:fig-s2}). This SNP is in the intron of a long intergenic noncoding gene, \emph{LINC00607}, that  is expressed in aorta, coronary, and tibial artery, all tissues potentially relevant to CIMT and atherosclerosis \cite{Consortium2015}. A maternally inherited allele at rs574232282 (G) in the intron of \emph{SCMH1} on chromosome 1 was associated with increased LVMI (P = 1.39 x10-8) (Supplementary Figure \ref{fig:fig-s3}). \emph{SCMH1} is expressed in aorta, coronary, and tibial artery \cite{Consortium2015}. \emph{SCMH1} protein associates with the polycomb group multiprotein complexes required to maintain the transcriptionally repressive state of certain genes\cite{Consortium2015}. Lastly, maternally inherited alleles at rs9849387 (A) and rs6791779 (C) on chromosome 3 were both associated with reduced FEV1 (P= 4.10x10-9 and 1.48x10-8, respectively) (Supplementary Figure \ref{fig:fig-s4}). The nearest gene to rs9849387 is \emph{ROBO2} (65kb, downstream), which is expressed in the lung as well as in brain, and ovary\cite{Consortium2015}.The nearest gene to rs6791779 is \emph{MIR4444-1}(267kb) whose expression has not been characterized.

\begin{figure}[!htb]
\centering \includegraphics[width=6in]{img/ch02/fig-01-menarche_matpatgwas}
\caption[Maternal and Paternal GWAS results for Age of Menarche.]{\textbf{Maternal and Paternal GWAS results for Age of Menarche.}  The top panel shows the Manhattan plots from the maternal (left) and maternal (right) GWAS. LocusZoom plots for both GWAS are shown in the lower panel for the associated region in the GWAS. Boxplots show the distribution of age of menarche residuals (y-axes) by the corresponding maternal and paternal alleles at this SNP (x-axes). The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:menarche_mpgwas}
\end{figure}

Three other CVD-related phenotypes (systolic blood pressure, LDL-C, and total cholesterol) had associations with paternally inherited alleles only. The paternally inherited allele at rs12024326 (A) on chromosome 1 was associated with lower LDL-cholesterol levels (P = 8.06x10-10) (Figure \ref{fig:ldl_mpgwas}). rs12024326 is in the intron of gene \emph{ADCK3}, and the same allele was associated with increased expression of \emph{ADCK3} in whole blood, as well as decreased expression of a neighboring gene, \emph{CDC42BPA} in brain (cerebellum), heart (left ventricle), esophagus, and tibial artery\cite{Consortium2015}.The paternal G allele at rs4843650 on chromosome 16 was associated with increased LDL-C and is located in the intron of \emph{JPH3}, which is expressed predominantly in the brain\cite{Consortium2015}. A SNP on chromosome 13 (rs1536182) was associated with systolic blood pressure levels when it was inherited from the father (Figure S5). The paternally inherited A allele at this SNP was associated with decreased systolic blood pressure, as well as decreased expression of its closest gene, \emph{LINC01055}, a long intergenic noncoding gene, in testis\cite{Consortium2015}. A paternally inherited allele at rs113588203 (G) on chromosome 1 was associated with lower total cholesterol (P = 1.76x10-8) (Figure S6). This SNP is intergenic between \emph{RHOU} (96kb, downstream), which is expressed across multiple tissues, and \emph{MIRR4454} (331kb), which is expressed in adipose, kidney and heart tissues\cite{Consortium2015}. 


\begin{figure}[!htb]
\centering \includegraphics[width=6in]{img/ch02/fig-02-ldl_matpatgwas}
\caption[Maternal and Paternal GWAS results for LDL Cholesterol.]{\textbf{Maternal and Paternal GWAS results for LDL Cholesterol.}  The top panel shows the Manhattan plots from the maternal (left) and maternal (right) GWAS. LocusZoom plots for both GWAS are shown in the lower panel for the associated region in the GWAS. Boxplots show the distribution of LDL residuals (y-axes) by the corresponding maternal and paternal alleles at this SNP (x-axes). The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:ldl_mpgwas}
\end{figure}


\subsection{GWAS for Differential Parent of Origin Effects}\label{GWAS for Differential Parent of Origin Effects}

Because some imprinted regions include genes that have both maternal or paternal specific tissue expression, we next tested for such differential effects with these 21 phenotypes. In these analyses, we compared the effect and direction of the association between maternal and paternal alleles to identify variants that have different effects, including opposite effects, on the phenotype. Such loci would be completely hidden in standard GWAS in which paternally and maternally inherited alleles are combined. These opposite effect GWAS revealed 11 independent loci with opposite POEs for nine different traits, at least six of which are associated with CVD risk (Table 3, Figure S7). 

A locus on chromosome 16, near the \emph{CDH8} gene (128kb, upstream), was associated with opposite POEs with age of menarche (Figure \ref{fig:menarche_oegwas}). CDH8 is highly expressed in the brain, as well as in the aorta artery and pituitary gland. Two loci on chromosomes 5 and 6 were associated with opposite POEs on body mass index (BMI) (Figure \ref{fig:bmi_oegwas}). The most significant variant on chromosome 5 (rs77785972) is near a long intergenic noncoding gene, \emph{LINC01340} (409kb, downstream), whose expression has not been well characterized. The SNP on chromosome 6 (rs17605739) is also in a long intergenic noncoding gene, \emph{RP1-209A6.1}, which is expressed in low levels in the tibial artery, bladder, spleen, lung, pituitary gland, as well as testis. 


\begin{figure}[!htb]
\centering \includegraphics[width=5in]{img/ch02/fig-03-menarche_oegwas}
\caption[Opposite Effect Parent of Origin GWAS Result for Age of Menarche.]{\textbf{Opposite Effect Parent of Origin GWAS Result for Age of Menarche.}  Box plots of age of menarche residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal C allele is associated with decreased and maternal T allele with increased age of menarche. The paternal C allele is associated with increased and the paternal T allele with decreased age of menarche. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:menarche_oegwas}
\end{figure}

\begin{figure}[!htb]
\centering \includegraphics[width=5in]{img/ch02/fig-04-bmi_oegwas}
\caption[Opposite Effect Parent of Origin GWAS Result for BMI.]{\textbf{Opposite Effect Parent of Origin GWAS Result for BMI.}  Box plots of two significant loci plot BMI residuals (y-axes) for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. For the (A) SNP on chromosome 5 the maternal A allele is associated with decreased and maternal G allele with increased BMI. The paternal A allele is associated with increased and the paternal G allele with decreased BMI. For the (B) SNP on chromosome 6 the maternal A allele is associated with decreased and maternal G allele with increased BMI. The paternal A allele is associated with increased and the paternal G allele with decreased BMI. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:bmi_oegwas}
\end{figure}



 A SNP on chromosome 16 (rs1032596) was associated with opposite POEs on LDL-cholesterol (Figure S8). This SNP lies in the intron of another long noncoding RNA gene, \emph{LINC01081}, which has been suggested to be imprinted because its downstream genes have also been shown to have parent- and tissue-specific activity\cite{Szafranski:2016fz}. A region on chromosome 2 has opposite effects associated with LVMI (Figure S9). The associated SNPs are in the intron of \emph{XIRP2}, a cardiomyopathy associated protein that is expressed in skeletal muscle and heart left ventricle, suggesting that this gene could play a role in determining left ventricular mass\cite{Wang:2014jg,Nilsson:2013jy,Consortium2015}. In addition, the most significant SNP at this region, rs17616252 (and multiple SNPs in LD) is a strong eQTL (P = 1.8x10-13) for the gene \emph{XIRP2} in skeletal muscle, \emph{XIRP2-AS1} in testis, and \emph{B3GALT1} in transformed fibroblast cells\cite{Consortium2015}. Four variants in a region on chromosome 1 in a microRNA gene, \emph{MIR548F3}, were associated with opposite POEs on triglyceride levels (Figure S10). The expression of \emph{MIR548F3} has not been characterized. SNP rs7033776 near \emph{MELK} (27kb, downstream) on chromosome 9 was associated with opposite effects on total cholesterol (Figure S11). \emph{MELK} is expressed in the colon and esophagus in addition to transformed lymphocytes and fibroblasts\cite{Consortium2015}. 


Nine linked variants on chromosome 1 were associated with opposite POEs of blood eosinophil count (Figure S12). These variants are near the gene \emph{IGSF21} (27kb, downstream) which is a member of the immunoglobulin superfamily and likely acts as a receptor in immune response pathways\cite{OLeary:2016cm}. A variant on chromosome 3, rs12714812, was associated with opposite POEs for FEV1 (Figure S13). This variant has been shown to regulate the expression of a gene \emph{CNTN3} (45kb, upstream) in heart and brain\cite{Consortium2015}. Studies in mice have suggested that this gene is imprinted and maternally expressed in the murine placenta\cite{Brideau:2010gz}. Variant rs142030841 in the intron of the gene TPGS2 on chromosome 18 has opposite POEs with neutrophil levels (Figure S14).¬ This SNP is an expression quantitative trait locus (eQTL) for the noncoding RNA gene \emph{RP11-95O2.5} in skin, testis, breast, thyroid and adipose tissue, for \emph{CELF4} in tibial nerve and lung, and for \emph{TPGS2} in tibial artery and transformed fibroblast cells\cite{Consortium2015}.


\subsection{Parent of Origin Effects on Gene Expression}\label{Parent of Origin Effects on Gene Expression}

To determine if any of the associated variants also showed POEs on gene expression in the Hutterites, we used RNA-seq gene expression data from lymphoblastoid cell lines (LCLs) collected from 430 of the individuals in the GWAS sample. We first tested for association of maternal and paternal variants with genes detected as expressed in the LCLs and whose transcript start site was within 1Mb of each associated SNP. There were no significant associations after multiple testing correction, similar to a previous study\cite{Benonisdottir:2016dz}. However, because we considered this to be exploratory analyses, we show results for the five most significant parent of origin eQTLs (Table 3). We next used the opposite effect model for each SNP in Table 2 and expression of all genes that were detected as expressed in LCLs and whose transcript start site was within 1Mb of the associated SNP. This resulted in 57 tests (1 SNP for each of 8 phenotypes, and 57 genes). The five most significant opposite effect eQTLs, none of which passed the Bonferroni threshold of 8.77 x 10-4, are shown in Table 4. The most significant opposite effect eQTL was for POLR1E expression with the SNP on chromosome 9 (rs7033776) that was associated with total cholesterol (opposite effect eQTL P = 9.86 x 10-4) (Figure S15). POLR1E is involved in the purine metabolism pathway as well as DNA-directed polymerase activity. The same SNP, rs7033776, had modest opposite effects with the expression of three other genes in the region (PAX5, FBXO10, and FRMPD1), a signature consistent with an imprinted region. Another SNP with opposite POEs on LVMI, rs16853098, was an opposite effect eQTL for STK39, a gene that has been previously associated with hypertension\cite{Wang:2009bt}.


\section{Discussion}\label{fig-discussion}

In this study, we introduced a novel statistical method that allows assessment of standard GWAS signals along with measures of differential POEs on common quantitative phenotypes. Similar to previous parent of origin studies of fewer phenotypes\cite{Kong:2009kk,Benonisdottir:2016dz,Garg2012a}, we tested for associations of maternally- or paternally-derived alleles with each phenotype. We then extended this method to identify variants for which maternally- and paternally-derived alleles have different, including opposite, effects on phenotypic values. The focus on 21 common disease-associated phenotypes in a single large pedigree allowed us to broadly survey physiological effects of putative imprinted regions and the candidate genes at each associated locus. In contrast to previous studies, our new model can identify variants with opposite POEs that would be missed in traditional GWAS (Table 2). 

Our studies of \textgreater1,000 Hutterites who are related to each other in a single pedigree allowed us to detect POEs, even when few genome-wide significant associations were detected in standard GWAS of the same phenotypes. Our method revealed parent of origin specific genome-wide significant associations for seven of the 21 phenotypes examined, with maternally-inherited alleles associated with four phenotypes, paternally-inherited alleles with three phenotypes (Table 1), and opposite parent of origin alleles with nine phenotypes, of which five also showed single POEs at different loci (Table 2). Overall, 11 of the 21 phenotypes examined showed genome-wide significant evidence of POEs with alleles at one or more loci. In contrast, standard GWAS of these same phenotypes and using the same markers in these same individuals revealed genome-wide significant association for only five traits. 

It is notable that four of the nine significant opposite parent of origin effects (one each with LDL-C and triglycerides, and two with BMI) lie in or near long intergenic non-coding RNA genes (lincRNAs). LincRNAs are a feature of imprinted regions\cite{Peters2014}, where they can silence the expression of genes on the opposite chromosome\cite{Barlow:2014dv,Patten:2016cb}. One of the variants, rs1032596, with an opposite parent of origin effect on LDL-C is located in the \emph{LINC01081} gene. This noncoding RNA, along with \emph{LINC01082}, regulates the \emph{FOXF1} enhancer resulting in \emph{FOXF1} parent- and tissue-specific activity\cite{Szafranski:2016fz} providing experimental support for tissue specific expression, a feature of imprinted regions. 

Another variant with POEs in our study has been suggested to be imprinted in previously published work. The variant associated with opposite POEs for FEV1 is an eQTL for the gene \emph{CNTN3}. \emph{CNTN3} was shown to have exclusive maternal allele-specific expression in murine placentas\cite{Brideau:2010gz}, although this finding may have been due to contaminating maternal cells\cite{Okae:2011hj,Proudhon:2011eh}. 

Other regions associated with POEs harbor genes involved in transcriptional repression (e.g., \emph{SCMH1} with LVMI on chromosome 1) or the associated SNPs are reported as eQTLs in GTEx with expression in tissues relevant to the phenotype under investigation (e.g., the LVMI-associated SNPs are eQTLs for \emph{XIRP2}, which is expressed in skeletal muscle and heart left ventricle)\cite{Consortium2015}. Overall, these patterns of expression provide additional support that the parent of origin associations in our study are flagging imprinted regions or regions involved in the regulation of gene expression. Finally, we used gene expression in LCLs from the Hutterites to directly test for parent of origin eQTLs among SNPs associated with phenotypes in the parent of origin GWAS. Although none of the parent of origin eQTLs met criteria for significance after correcting for multiple testing, the SNP on chromosome 9 with opposite POEs on total cholesterol levels was borderline significant as an opposite parent of origin eQTL for \emph{POLR1E}, and possible for three other genes at the same locus (\emph{PAX5}, \emph{FBXO10}, and \emph{FRMPD1}). The presence of multiple genes with potential parent of origin expression patterns is further supportive of an imprinted locus. The availability of gene expression only in LCLs from the Hutterites limits the inferences we can draw about effects on expression because imprinted regions are often tissue-specific and sometimes developmentally regulated\cite{Peters2014,Falls1999}. Despite this limitation, the fact that many of the SNPs associated with POEs on phenotypes are themselves eQTLs in relevant tissues (GTEx) and some are suggestive of having POEs on expression in LCLs from the Hutterites is generally supportive of the suggestion that some of the regions identified in this study are imprinted or have network interactions with imprinted genes\cite{Chess:2016fs} in humans. Additionally, our data suggest that loci with POEs influence a broad spectrum of quantitative phenotypes that are themselves risk factors for common diseases. 

 In particular, the discovery of POEs for eight traits that are associated cardiovascular disease risk is intriguing. These include metabolic phenotypes, such as BMI, total cholesterol, triglycerides, LDL, and age of menarche, that have indirect effects on cardiac health, as well as LVMI and CIMT, which more directly reflect cardiac health. Some of these phenotypes showed associations with paternally inherited alleles only (systolic blood pressure, LDL-C, total cholesterol), maternally inherited alleles only (LVMI, CIMT, and age at menarche), and/or with opposite effect variants (BMI, LDL-C, triglycerides, total cholesterol, LVMI, age at menarche). It has been suggested that genomic imprinting evolved in the mammalian lineage as a way to regulate maternally and paternally expressed genes in the placenta during pregnancy and modulate metabolic functions related to growth, where the parental interests may be in conflict ? paternal alleles favoring growth of the fetus at the expense of the mother while maternal alleles favor restricting resources to the fetus to ensure preservation of her nutritional needs \cite{Haig:2000if,Barlow:2014dv,Patten:2016cb}. Our data show some effects that are consistent with this theory. For example, three independent paternally inherited alleles on chromosome 1 are associated with increased LDL-C (Fig. 2) and total cholesterol (Figure S7); a paternal allele on chromosome 13 is also associated with increased systolic blood pressure (Figure S6). However, it is not always possible to interpret our results in light of this model, such as the association of maternal allele on chromosome 2 with decreased CIMT (Figure S3), or the maternal allele on chromosome 16 associated with decreased age of menarche (Figure  \ref{fig:menarche_mpgwas}), which confers increased cardiovascular risk \cite{Canoy:2015ha}. However, because many of the traits associated with POEs in this study were measured in adults, and none were measured in neonates, we are likely observing the downstream effects of processes that occurred in utero. Nonetheless, this kinship theory, or parent-conflict hypothesis, could account for the enrichment of parent of origin associations, particularly those with opposite effects, among metabolic and CVD-associated traits \cite{Peters2014}.
 
 Finally, we note that the parent of origin GWAS for 21 phenotypes in the Hutterites revealed overall twice as many genome-wide significant loci compared to standard GWAS of the same phenotypes in the same individuals, suggesting that variation at imprinted loci may represent some of the "missing heritability" of these phenotypes and potentially for the disease for which they confer risk. This idea is consistent with observations in both mice and humans \cite{Laurin:2017jv}. POEs in mice contribute disproportionally to the heritability of 97 traits, including those related to total cholesterol, weight, HDL, and triglycerides \cite{Mott2014}. Exactly how much loci with POEs in humans contribute to phenotypic variation and disease risk overall remains to be determined, but our study provides compelling evidence that it is likely to be significant for many important traits.


\section{Methods}\label{fig-methods}

\subsection{Sample Composition}\label{Sample Composition}
The individuals in this study have participated in one or more of our studies on the genetics of complex traits in the Hutterites \cite{Cusanovich:2016id,Weiss:2005cq,Abney2001}. The more than 1,500 Hutterites in our study are related to each other in a 13-generation pedigree including 3,671 individuals. 

\subsection{Genotype Data}\label{Genotype Data}
Variants detected in the whole genome sequences of 98 Hutterites were previously imputed to an additional 1,317 individuals using PRIMAL, a high-accuracy pedigree based imputation method \cite{Livne2015}. PRIMAL was used to phase alleles and assign parent of origin for 83\% of about 12 million autosomal SNPs. For these studies, we selected SNPs that had a MAF 1\% and genotype call rate 85\%. This yielded 5,891,982 autosomal SNPs. Parent of origin allele call rates differed among individuals and between phenotypes (Table S1 %\ref{tab:fig-s1}).

\subsection{Phenotype Data}\label{Phenotype Data}
We included 21 quantitative phenotypes that were previously measured in the Hutterites. Descriptions for each phenotype, as well as exclusion criteria, transformations, and covariates used with each phenotype in the GWAS, are available in the Supplementary Methods (Table S1 %\ref{tab:fig-s1}). 

Descriptions for 18 of the 21 phenotypes can be found in Cusanovich et al \cite{Cusanovich:2016id}. The remaining three are described here. Height was measured in cm on a stadiometer with shoes removed. BMI was calculated using weight (kg, measured on scale) divided by height (m) squared. Age at menarche was collected retrospectively by interview. 

\subsection{GWAS}\label{GWAS Methods}
We used a linear mixed model as implemented in GEMMA to test for genome wide association with 21 phenotypes using an additive model. We corrected for relatedness, as well as relevant covariates (Table S1 
%\ref{tab:fig-s1}).


\subsection{Maternal and Paternal GWAS}\label{Maternal and Paternal GWAS Methods}
To evaluated the evidence for POEs, we tested maternal and paternal alleles separately with each phenotype, comparing phenotypic differences between the maternally inherited alleles and between the paternally inherited alleles. We used a linear mixed model as implemented in GEMMA, which allows us to correct for relatedness as a random effect, as well as sex, age, and other covariates as fixed effects \cite{Zhou2012}.The linear mixed model for the parent of origin GWAS for testing maternal alleles and paternal alleles is shown in Equation 2.1 and Equation 2.2, respectively. 

\begin{equation}
Y =W\alpha + X_{M}\beta_{M}+g+\epsilon
\end{equation}

\begin{equation}
Y =W\alpha + X_{P}\beta_{P}+g+\epsilon
\end{equation}

$n$ is the number of individuals, $Y$ is an $n$ × 1 vector of quantitative traits, $W$ is an $n × c$ matrix of covariates (fixed effects) including intercept 1. $\alpha$ is a $c$ × 1 vector of covariate coefficients. $X_M$  is an $n$ × 1 vector of maternal alleles, and $X_P$ an $n$ × 1 vector of paternal alleles. $\beta_M$ and $\beta_P$ are the effect sizes of maternal and paternal alleles, respectively. g is a vector of genetic effects with $g \sim N(0, A(\sigma_g)^2 )$ where A is the genetic relatedness matrix; $\epsilon$ is a vector of non-genetic effects with $\epsilon \sim N(0,I(\sigma_e)^2)$.

\subsection{Differential Effect GWAS (PO-GWAS)}\label{Differential Effect GWAS (PO-GWAS)}
To test for a difference in the same allele inherited from each parent, including opposite effects, we re-parameterized the test model (Equation 2.3) from Garg et al\cite{Garg2012a}. The null model (Equation 2.4) is a standard GWAS model, ignoring parent of origin of alleles. The test model (Equation 2.3) is more significant when maternal and paternal alleles have differential effects on gene expression.

\begin{equation}
Y =W\alpha + X_{M}\beta_{M}+ X_{P}\beta_{P}+g+\epsilon
\end{equation}

This new model allows us to measure the difference in parental effect of the same allele when the genotype is a covariate in Equation 2.5. 
\begin{equation}
Y =W\alpha + X_{MP}\beta_{MP}+g+\epsilon
\end{equation}

$X_{PM}$ is a $n$ × 1 vector of genotypes with possible values [ 0,1,2 ], equivalent to $X_P+ X_M. (\beta_M-\beta_P )$ is the difference in parental effect size. If the difference in parental effect size is large and significantly different from 0 it suggests a parent of origin effect exists at this variant. $((X_M-X_P ))/2$ is a $n$ × 1 vector of genotypes with possible values [-1,0,1].  $((\beta_P+\beta_M))/2$   is the average parental effect size that is captured in normal GWAS using genotypes. The average genotypes are added in as a covariate, with the average parental effect size the corresponding covariate coefficient. This differential effect GWAS was tested in GEMMA using BIMBAM format to use average genotype values\cite{Servin:2007gj}.


\subsection{Parent of Origin eQTL studies}\label{Parent of Origin eQTL studies}

RNA-seq data from LCLs were available from a previous study in the Hutterites\cite{Cusanovich:2016id}. For this study, sequencing reads were reprocessed as follows. Reads were trimmed for adaptors using Cutadapt (with reads \textless5 bp discarded) then remapped to hg19 using STAR indexed with gencode version 19 gene annotations\cite{Dobin:2002by,Martin:2011eu}. To remove mapping bias, reads were processed using WASP mapping pipeline\cite{vandeGeijn:2015hi}. Gene counts were collected using HTSeq-count\cite{Anders:2015gf}. VerifyBamID was used to identify sample swaps to include individuals that were previously excluded\cite{Jun:2012je}. Genes mapping to the X and Y chromosome were removed; genes with a Counts Per Million (CPM) value of 1 (expressed with less than 20 counts in the sample with lowest sequencing depth) were also removed. Limma was used to normalize and convert counts to log transformed CPM values \cite{Ritchie:2015fa}. Technical covariates that showed a significant association with any of the top principal components were regressed out (RNA Integrity Number and RNA concentration).



\subsection{Maternal and Paternal Parent of Origin eQTL}\label{Maternal and Paternal Parent of Origin Parent of Origin eQTL}

LCL RNA-seq data was used to test the single parent model for the most significant SNP from the maternal or paternal only GWAS for each phenotype. We selected all genes detected as expressed in the LCLs and residing within 1Mb of each most significant associated SNP. Summary of the SNPs and genes tested are in Table S3.

\subsection{Differential Parent of Origin eQTL}\label{Differential Parent of Origin eQTL}

LCL RNA-seq data was used to test the opposite effect model for the most significant SNP in each region that was associated with a phenotype in the parent of origin opposite effects GWAS. We selected all genes detected as expressed in the LCLs and residing within 1Mb of each associated SNP. Summary of the SNPs and genes tested are in Table S4.

\section{Supplementary Information}\label{fig-supplementary-information}

\subsection{Supplementary Figures}\label{fig-supplementary-figures}
\clearpage

\renewcommand\theContinuedFloat{\alph{ContinuedFloat}}
 
\begin{figure}[!htb]
\ContinuedFloat*
	\centering
		\includegraphics[width=5in]{img/ch02/fig-s1a.pdf}
	\caption[Manhattan and QQ Plots from Standard GWAS of 21 Quantitative Phenotypes.]{\textbf{Manhattan and QQ Plots from Standard GWAS of 21 Quantitative Phenotypes.} }
	\label{fig:fig-s1a}
\end{figure}

\begin{figure}[!htb]
	\ContinuedFloat
	\centering
		\includegraphics[width=5in]{img/ch02/fig-s1b.pdf}
\caption[Manhattan and QQ Plots from Standard GWAS of 21 Quantitative Phenotypes (Continued).]{\textbf{Manhattan and QQ Plots from Standard GWAS of 21 Quantitative Phenotypes (Continued).} }
\label{fig:fig-s1b}
\end{figure}

 
\begin{figure}[!htb]
\ContinuedFloat
\centering
\includegraphics[width=5in]{img/ch02/fig-s1c.pdf}
\caption[Manhattan and QQ Plots from Standard GWAS of 21 Quantitative Phenotypes (Continued).]{\textbf{Manhattan and QQ Plots from Standard GWAS of 21 Quantitative Phenotypes(Continued).} }
\label{fig:fig-s1c}
\end{figure}




\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s2.pdf}
\caption[Maternal and Paternal GWAS results for CIMT.]{\textbf{Maternal and Paternal GWAS results for CIMT.}  The top panel shows the Manhattan plots from the paternal (left) and maternal (right) GWAS. LocusZoom plots are shown in the lower panel for the associated region in the GWAS. Boxplots show the distribution of CIMT residuals (the residuals correspond to the inverse of raw CIMT values) (y-axes) by the corresponding maternal and paternal alleles at this SNP (x-axes). The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s2}
\end{figure}



\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s3.pdf}
\caption[Maternal and Paternal GWAS results for LVMI.]{\textbf{Maternal and Paternal GWAS results for LVMI.}  The top panel shows the Manhattan plots from the paternal (left) and maternal (right) GWAS. LocusZoom plots are shown in the lower panel for the associated region in the GWAS. Boxplots show the distribution of LVMI residuals (the residuals correspond to the inverse of raw CIMT values) (y-axes) by the corresponding maternal and paternal alleles at this SNP (x-axes). The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s3}
\end{figure}



\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s4.pdf}
\caption[Maternal and Paternal GWAS results for FEV1.]{\textbf{Maternal and Paternal GWAS results for FEV1.}  The top panel shows the Manhattan plots from the paternal (left) and maternal (right) GWAS. LocusZoom plots for both GWAS are shown in the lower panel for the associated region in the GWAS. Boxplots show the distribution of FEV1 residuals (y-axes) by the corresponding maternal and paternal alleles at this SNP (x-axes). The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show 1.5 x IQR.}
\label{fig:fig-s4}
\end{figure}




\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s5.pdf}
\caption[Maternal and Paternal GWAS results for Systolic Blood Pressure.]{\textbf{Maternal and Paternal GWAS results for Systolic Blood Pressure.}  The top panel shows the Manhattan plots from the paternal (left) and maternal (right) GWAS. LocusZoom plots for both GWAS are shown in the lower panel for the associated region in the GWAS. Boxplots show the distribution of systolic blood pressure residuals (y-axes) by the corresponding maternal and paternal alleles at this SNP (x-axes). The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show 1.5 x IQR.}
\label{fig:fig-s5}
\end{figure}




\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s6.pdf}
\caption[Maternal and Paternal GWAS results for Total Cholesterol.]{\textbf{Maternal and Paternal GWAS results for Total Cholesterol.}  The top panel shows the Manhattan plots from the paternal (left) and maternal (right) GWAS. LocusZoom plots for both GWAS are shown in the lower panel for the associated region in the GWAS. Boxplots show the distribution of total cholesterol residuals (y-axes) by the corresponding maternal and paternal alleles at this SNP (x-axes). The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show 1.5 x IQR.}
\label{fig:fig-s6}
\end{figure}



\begin{figure}[!htb]
	\ContinuedFloat*
	\centering
	\includegraphics[width=5in]{img/ch02/fig-s7a.pdf}
	\caption[Manhattan and QQ Plots from Differential Effect GWAS of 21 Quantitative Phenotypes]{\textbf{Manhattan and QQ Plots from Differential Effect GWAS of 21 Quantitative Phenotypes} }
	\label{fig:fig-s7a}
\end{figure}


\begin{figure}[!htb]
	\ContinuedFloat
	\centering
	\includegraphics[width=5in]{img/ch02/fig-s7b.pdf}
	\caption[Manhattan and QQ Plots from Differential Effect GWAS of 21 Quantitative Phenotypes (Continued). ]{\textbf{Manhattan and QQ Plots from Differential Effect GWAS of 21 Quantitative Phenotypes (Continued).} }
	\label{fig:fig-s7b}
\end{figure}



\begin{figure}[!htb]
	\ContinuedFloat
	\centering
	\includegraphics[width=5in]{img/ch02/fig-s7c.pdf}
	\caption[Manhattan and QQ Plots from Differential Effect GWAS of 21 Quantitative Phenotypes (Continued). ]{\textbf{Manhattan and QQ Plots from Differential Effect GWAS of 21 Quantitative Phenotypes (Continued).} }
	\label{fig:fig-s7c}
\end{figure}


\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s8.pdf}
\caption[Opposite Effect Parent of Origin GWAS Result for LDL.]{\textbf{Opposite Effect Parent of Origin GWAS Result for LDL.}  Box plots of LDL residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal C allele is associated with decreased and maternal T allele with increased LDL. The paternal C allele is associated with increased and the paternal T allele with decreased LDL. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s8}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s9.pdf}
\caption[Opposite Effect Parent of Origin GWAS Result for LVMI.]{\textbf{Opposite Effect Parent of Origin GWAS Result for LVMI.}  Box plots of LVMI residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal C allele is associated with decreased and maternal T allele with increased LVMI. The paternal C allele is associated with increased and the paternal T allele with decreased LVMI. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s9}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s10.pdf}
\caption[Opposite Effect Parent of Origin GWAS Result for Triglycerides.]{\textbf{Opposite Effect Parent of Origin GWAS Result for Triglycerides.}  Box plots of triglyceride residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal C allele is associated with decreased and maternal T allele with increased triglycerides. The paternal C allele is associated with increased and the paternal T allele with decreased triglycerides. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s10}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s11.pdf}
\caption[Opposite Effect Parent of Origin GWAS Result for Total Cholesterol.]{\textbf{Opposite Effect Parent of Origin GWAS Result for Total Cholesterol.}  Box plots of total cholesterol residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal C allele is associated with decreased and maternal T allele with increased total cholesterol. The paternal C allele is associated with increased and the paternal T allele with decreased total cholesterol. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s11}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s12.pdf}
\caption[Opposite Effect Parent of Origin GWAS Result for Blood Eosinophil Count.]{\textbf{Opposite Effect Parent of Origin GWAS Result for Blood Eosinophil Count.}  Box plots of eosinophil residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal C allele is associated with decreased and maternal T allele with increased eosinophil count. The paternal C allele is associated with increased and the paternal T allele with decreased eosinophil count. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s12}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s13.pdf}
\caption[Opposite Effect Parent of Origin GWAS Result for FEV1.]{\textbf{Opposite Effect Parent of Origin GWAS Result for FEV1.} Box plots of FEV1 residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal C allele is associated with decreased and maternal T allele with increased FEV1. The paternal C allele is associated with increased and the paternal T allele with decreased FEV1. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s13}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s14.pdf}
\caption[Opposite Effect Parent of Origin GWAS Result for Neutrophil Count.]{\textbf{Opposite Effect Parent of Origin GWAS Result for Neutrophil Count.}  Box plots of neutrophil residuals (y-axes) are shown for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal G allele is associated with decreased and maternal Aallele with increased neutrophil count. The paternal G allele is associated with increased and the paternal A allele with decreased neutrophil count. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s14}
\end{figure}






\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/fig-s15.pdf}
\caption[Opposite Effect eQTL for rs7033776.]{\textbf{Opposite Effect eQTL for rs7033776.} Box plots of two significant loci plot POLR1E gene expression residuals (y-axes) for each of the four genotypes (left panel; x-axis), and for paternal (center panel; x-axis) and maternal (right panel; x-axis) alleles. The maternal T allele is associated with decreased and maternal A allele with increased POLR1E expression. The paternal T allele is associated with increased and the paternal A allele with decreased POLR1E expression. The horizontal bar of the boxplot shows the median, the box delineates the first and third quartile, and the whiskers show +/-1.5 x IQR.}
\label{fig:fig-s15}
\end{figure}













