% !TEX encoding = UTF-8 Unicode
\chapter{Conclusion}

As we look past the "low hanging fruit" of GWAS, we turn to other biological mechanisms where genetics can influence our traits, including studies or rare variation\cite{Igartua:2017ir}, gene by environment interactions, and parent of origin effects among others. The Hutterites that the Ober Lab studies are an ideal population to look for parent of origin effects on quantitative disease related traits and gene expression\cite{Weiss:2005cq,Abney2001,Ober:2001dy}. With RNA-seq and novel methods for imputing and phasing data at a population scale\citep{Livne2015}, we are able to start getting at these questions. 


\section{A novel method to detect opposite effects of parentally inherited variants on cardiovascular disease and asthma associated traits}
 
 In Chapter \ref{ch:pogwas}, I describe our study to detect genetic variation than can impact cardiovascular disease and asthma associated traits when inherited from one parent and not the other. In order to maximize on our data, we develop a novel method in this chapter that can be used to detect if parentally inherited alleles can have opposite effects on quantitative traits depending on which parent they were inherited from. These associations would be missed in traditional GWAS. Previous studies have explored parent of origin effects but mostly using trios\cite{Garg2012a,Ainsworth:2010bp,Howey:2012hj} and on case control status\cite{Kong:2009kk,Ainsworth:2010bp}. Only a few have begun to search for parent of origin effects on quantitative traits, starting with height\cite{Benonisdottir:2016dz,Zoledziewska:2015do}.
 
Using this novel method, we identify parent of origin effects (POEs) on 11 phenotypes in the Hutterites, most of which are risk factors for cardiovascular disease. Most of the loci we identified have features of imprinted regions and many of the variants are associated with expression of nearby genes. Overall, we show that POEs which can be opposite in direction, are relatively common in humans and have potentially important clinical effects. 
 
 \subsection{Future Directions}
It is necessary to still replicate the results in a different population to verify these effects exist. They did not replicate in the Sardinia population. Additionally a power analysis should be done to confirm our method would pick up genetic variants of the effect size and minor allele frequency that we identified.

It is important to further investigate these regions. One of the regions with a POE identified in our study, \emph{LINC01081} has been studied in detail by another group showing that a parent of origin effect exists at this region independent of our study\cite{Szafranski:2016fz}. Others have shown POEs at known imprinted regions that affect height\cite{Benonisdottir:2016dz,Zoledziewska:2015do}. It would be the next step to show that the regions we identify with POEs could be at imprinted regions.
 
\section{Identifying two novel imprinted genes in known imprinted regions using parent of origin gene expression}

 In Chapter \ref{ch:imprinted}, I identified two new imprinted genes using parent of origin and allele specific expression. Using a novel method as a variation on WASP\cite{vandeGeijn:2015hi}, I mapped RNA-seq reads to parental haplotypes using SNPs in the reads and parent of origin information of SNPs. We identified known imprinted genes and two novel imprinted genes, \emph{PXDC1} and \emph{PWAR6}, among our genes with asymmetrical parent of origin gene expression. We validated the patterns of gene expression using RNA-seq from peripheral blood leukocytes (PBLs). To validate the imprinted genes further, we used DNA methylation levels in the PBLs to confirm imprinting control regions (ICRs) previously defined\cite{Joshi:2016bb,Court:2014kc}. Our two new imprinted genes lie in known imprinted regions with known ICRs providing more evidence for their imprinting status.
 
This is the largest pedigree based genome-wide scan for imprinted regions to date with 306 Hutterite individuals. We also provide a new way of mapping reads to parent of origin haplotypes and identify two new possible imprinted genes.

 \subsection{Future Directions}
It is still necessary to further validate \emph{PXDC1} and \emph{PWAR6} as imprinted genes in a different population. Further characterization of this locus is still required. We identified these genes as imprinted but had overall very few parentally mapped reads. This would need to be replicated with many more parentally mapped reads. It is surprising they have not yet been discovered as imprinted genes since they are in known and pretty well characterized imprinted regions. It is possible they are tissue and developmentally specific as has been shown for other imprinted genes\cite{Baran:2015cx}. 

\section{Can parentally inherited genetic variation influence gene expression on the same haplotype?}

In Chapter \ref{ch:poeqtl}, I explored if parentally inherited genetic variation can affect gene expression as well as haplotype specific gene expression. Using the method of Chapter \ref{ch:pogwas} to detect opposite effect association, we did not identify any SNPs that had opposite effects on total gene expression depending on which parent the allele as inherited from. To get at the same effect in a different way, we performed a maternal eQTL (mat-eQTL) with maternal alleles on maternal expression as well as a paternal eQTL (pat-eQTL) with paternal alleles on paternal expression. Across all the SNP gene pairs run on both we did not identify any effects that were opposite in direction. 

To determine if any alleles had a specific effect on gene expression when inherited from one parent or the other, we used a parent of origin allele specific expression (PO-ASE) test and identified alleles that when inherited from one parent, and not the other, had a different effect on gene expression on that haplotype. 


 \subsection{Future Directions}
 
 While our model has picked up genes that look like they have a PO allele specific effect, there is still a lot to improve on. We use gene expression counts but our model is not modeling overdispersion which needs to be accounted for when looking at gene expression. We are looking or PO-ASE effects with very few parentally mapped reads. For standard eQTL studies we require at least 10 million reads, whereas, here, we have on average 1.8\% of 10 million reads mapped to a parental haplotype resulting on average in about 1.8 million reads to use for an eQTL study. Our study is likely underpowered to identify SNPs that have opposite effects on gene expression depending on which parent it was inherited from. More reads are required to perform this eQTL study and find parent specific effects. 
 
 Even with these parent of origin eQTLs identified in the latter part of Chapter \ref{ch:poeqtl}, it is not clear what known biological mechanism could lead to such an effect. It is possible that the parentally inherited allele different from the rest could disrupt normal expression by inducing silencing of the gene in a chromosome specific manner. It is also possible that the opposite allele - from the same parent - results in abnormal expression of the allele. Both of these would result in a parent of origin effect on gene expression but why or how these would exist is still not yet known. 

\section{Concluding remarks}




